devices/Adxl343/ADXL343/ADXL343.cs

```diff
@@ -0,0 +1,1748 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using nanoFramework.Hardware.Esp32;
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Numerics;
+using System.Threading;
+
+namespace ADXL343
```

Comment:
Please have a look at the other devices like https://github.com/nanoframework/nanoFramework.IoT.Device/blob/develop/devices/Adxl345/Adxl345.cs
And please follow the same naming conventions: for the namespace, including the cast, for the class with the proper cast as well.